var/spack/repos/builtin/packages/r-rsamtools/package.py

```diff
@@ -53,3 +53,8 @@ class RRsamtools(RPackage):
 
     # this is not a listed dependency but is needed
     depends_on("curl")
+    depends_on("zlib")
```

Comment:
Would this be better:
```suggestion
    depends_on("zlib-api")
```
?